benchmark/smallbank-bench

```diff
@@ -0,0 +1,238 @@
+
+const workers = [];
+const concurrenciesPerProcess = divideConcurrency(numberOfConcurrency);
+
+for (let i = 0; i < concurrenciesPerProcess.length; i++) {
+  const worker = fork('./smallbank-bench-worker', [
+    concurrenciesPerProcess[i].toString(),
+    start.toString(),
+    properties,
+    numberOfAccount.toString(),
+    rampUpTime.toString(),
+    operations.join(','),
+  ]);
+
+  worker.on('message', (m) => {
+    if (m.counter) {
+      counter++;
+    }
+
+    if (m.error) {
+      errorCount++;
+    }
+
+    if (m.totalCounter) {
+      totalCounter += m.totalCounter;
+    }
+
+    if (m.latencyTotal) {
+      latencyTotal += m.latencyTotal;
+    }
+  });
+
+  workers.push(worker);
+}
+
+let from = start;
+(async () => {
+  while (true) {
+    const to = Date.now();
+    if (to >= end) {
+      for (const w of workers) {
+        w.kill();
+      }
+      break;
+    }
+
+    console.log(`${(counter * 1000) / (to - from)} tps`);
+    counter = 0;
+    from = Date.now();
+    await sleep(1000);
+  }
+})().then(() => {
+  const tps = (totalCounter * 1000) / (duration * 1000);
+  const averageLatency = latencyTotal / totalCounter;
+
+  console.log(`TPS: ${tps}`);
+  console.log(`Average-Latency(ms): ${averageLatency}`);
+  console.log(`Error-Counts: ${errorCount}`);
+  process.exit(0);
+});
+
+function sleep(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+function parseArgv() {
+  let properties = 'client.properties.json';
+  let numberOfAccount = 10000;
+  let numberOfConcurrency = 1;
+  let duration = 200;
+  let rampUpTime = 60;
+  let operations = [
+    'transact_savings',
+    'deposit_checking',
+    'send_payment',
+    'write_check',
+    'amalgamate',
+  ];
+
+  const argv = process.argv.slice(2);
+
+  for (let i = 0; i < argv.length; i++) {
+    switch (argv[i]) {
+      case '--properties':
+      case '--config':
+        if (argv[i + 1] && fs.existsSync(argv[i + 1])) {
+          properties = argv[i + 1];
+          i++;
+        } else {
+          usage();
+          process.exit(1);
+        }
+        break;
+
+      case '--num-accounts':
+        if (argv[i + 1] && !isNaN(argv[i + 1]) && parseInt(argv[i + 1]) > 0) {
+          numberOfAccount = parseInt(argv[i + 1]);
+          i++;
+        } else {
+          usage();
+          process.exit(1);
+        }
+        break;
+
+      case '--num-concurrencies':
+        if (argv[i + 1] && !isNaN(argv[i + 1]) && parseInt(argv[i + 1]) > 0) {
+          numberOfConcurrency = parseInt(argv[i + 1]);
+          i++;
+        } else {
+          usage();
+          process.exit(1);
+        }
+        break;
+
+      case '--duration':
+        if (argv[i + 1] && !isNaN(argv[i + 1]) && parseInt(argv[i + 1]) > 0) {
+          duration = parseInt(argv[i + 1]);
+          i++;
+        } else {
+          usage();
+          process.exit(1);
+        }
+        break;
+
+      case '--ramp-up-time':
```

Comment:
When I specified `0` for this, it still triggers the help. `0` is useful for testing, so it would be great if it is fixed!

Reply:
No problem.